index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Go Green - Summit 2022</title>
+    <meta name="description" content="Go Green - Summit 2022">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="stylesheets/styles.css">
+</head>
+
+<body>
+    <section class="top-section">
+        <div class="top-cover">
+            <nav class="inactive">
+                <a href="#close"><img src="images/bars.png" alt="Nav Button"></a>
+                <a href="index.html" class="logo"><img src="images/green_logo.png" alt="logo"></a>
+                <ul>
```

Comment:
- [ ] Hi, at the top of the page you are missing the dark header that has the social media links please implement this as it is part of the design and is required for the project.

![image](https://user-images.githubusercontent.com/18730106/152666732-3b48e03c-d96e-42a8-a038-744368566826.png)